worker/src/main/scala/com/lucidchart/piezo/WorkerTriggerListener.scala

```diff
@@ -1,42 +1,49 @@
 package com.lucidchart.piezo
 
-import com.lucidchart.util.statsd.StatsD
+import com.timgroup.statsd.StatsDClient
 import java.util.Properties
 import org.quartz.Trigger.CompletedExecutionInstruction
 import org.quartz._
 import org.slf4j.LoggerFactory
+import scala.util.Try
 
 object WorkerTriggerListener {
   private val logger = LoggerFactory.getLogger(this.getClass)
 }
 
-class WorkerTriggerListener(props: Properties, statsd: StatsD) extends TriggerListener {
+class WorkerTriggerListener(props: Properties, statsd: StatsDClient, useDatadog: Boolean) extends TriggerListener {
   val triggerHistoryModel = new TriggerHistoryModel(props)
-  val triggerMonitoringPriorityModel = new TriggerMonitoringModel(props)
+
   def getName: String = "WorkerTriggerListener"
 
   def vetoJobExecution(trigger: Trigger, context: JobExecutionContext): Boolean = false
 
-  def triggerFired(trigger: Trigger, context: JobExecutionContext):  Unit = {}
+  def triggerFired(trigger: Trigger, context: JobExecutionContext):  Unit = {
```

Comment:
Tested with a dummy job. The fired comes before the complete. I'll skip adding logging here because of the builtin trigger logging plugin that we can enable if we want.